README.md

```diff
@@ -109,13 +114,16 @@ file, replacing the inside of the quotes with your key/secret:
 
 ```
 export GITHUB_KEY="your_key"
-export GITHUB_SECRET="your_secret"
+export GITHUB_GITHUB_SECRET="your_secret"
+export CLOUDINARY_URL="cloudinary://your_key:your_secret"
```

Comment:
On adopting `.env` strategy, you could generate a `.env.sample` file with the variables, and replace the instructions 

```
 On Unix-based systems using the bash shell, add the following to your `~/.bashrc` 
 file, replacing the inside of the quotes with your key/secret:
```
for something like:
```
Copy `.env.sample` file to `.env` and fill appropriately the variables settings. 
```